pkg/rbac/rule.go

```diff
@@ -71,6 +71,15 @@ func (set RuleSet) FilterResource(res string) (out RuleSet) {
 	return
 }
 
+func (set RuleSet) FilterGenericRules() (out RuleSet) {
```

Comment:
can we use [copy](https://pkg.go.dev/builtin#copy)() here?

Reply:
There is more inside that func now, after a new commit :)